tailwind.config.ts

```diff
@@ -106,7 +107,23 @@ const config = {
       },
     },
   },
-  plugins: [require("tailwindcss-animate")],
+  plugins: [
+    require("tailwindcss-animate"),
+    plugin(function ({ matchVariant }) {
```

Comment:
This plugin creates a utility class for the `:not()` pseudo. It allows for either...

An arbitrary variant
```css
/* "not-[:checked]:bg-primary" */

.not-\[\:checked\]\:bg-primary:not(:checked) {
    background-color: var(--primary);
}
```

Or a specifically defined variant, such as `disabled` here
```css
/* not-disabled:bg-primary */

.not-disabled\:bg-primary:not(:disabled) {
    background-color: var(--primary);
}
```